README.md

```diff
@@ -284,30 +284,29 @@ The test doesn't require any special setup. The instructions to run it are insid
 
 ## Run Salesforce Mobile test
 
+- There are two sample tests under `force-app/test/mobile`: `navigation.spec.js` is against Community playground application and `login.spec.js` is against Salelsforce application.
 - Follow the instructions at [Get Started for Mobile](https://utam.dev/guide/get_started_utam#get-started-for-mobile) to set up your local simulator/emulator.
 - Make sure that [Appium](https://github.com/appium/appium#readme) and [node.js](https://nodejs.org/en/) are installed on your machine.
 - Update the wdio configuration file:
-For an iOS test, update `wdio.conf.ios.js` file to configure the test device name(appium:deviceName), iOS version(appium:platformVersion), and the full path for the test application(appium:app):
+For an iOS test, update `wdio.conf.xx.ios.js` file to configure the test device name(appium:deviceName), iOS version(appium:platformVersion), and the full path for the test application(appium:app):
```

Comment:
nit: maybe explain what `xx` stands for or present it in a very straightforward way. At the same time I am thinking about reorganizing the project in a monorepo so that's the tests are scoped by package which might makes things cleaner so not a big deal